### PR TITLE
feat: first step for implementing meta-cli

### PR DESCRIPTION
**Pull request by tk3fftk**
This PR provides interface of the meta-cli.
I wrote `screwdriver.yaml` and `meta.go` based on [launcher](https://github.com/screwdriver-cd/launcher).

The help message of meta-cli is following:
```
% go build -o meta                                                                                                                 
% ./meta                                                                                                                           
NAME:
   meta-cli - get or set metadata for Screwdriver build

USAGE:
   meta command arguments [options]

VERSION:
   0.0.0

COMMANDS:
     get      Get a metadata with key
     set      Set a metadata with key and value
     help, h  Shows a list of commands or help for one command

GLOBAL OPTIONS:
   --meta-space value  Location for saving meta temporarily (default: "/tmp/meta")
   --help, -h          show help
   --version, -v       print the version

COPYRIGHT:
   (c) 2017 Yahoo Inc.
```

Related: https://github.com/screwdriver-cd/screwdriver/issues/293

### CODE REVIEW

File: meta.go [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+		},
+		cli.StringFlag{
+			Name:   "token",
+			Usage:  "JWT used for accessing Screwdriver's API",
+			EnvVar: "SD_TOKEN",
+		},
+		cli.StringFlag{
+			Name:  "meta-tmp",
+			Usage: "Location for saving meta temporarily",
+			Value: "/tmp/meta",
```
Comment (tk3fftk) [inline meta.go]: We are planning to use `/tmp/meta` directory for saving meta temporarily.

File: meta.go [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+	if VERSION == "" {
+		VERSION = "0.0.0"
+	}
+	app.Version = VERSION
+
+	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "meta-space",
+			Usage: "Location for saving meta temporarily",
+			Value: "/tmp/meta",
```
Comment (tk3fftk) [inline meta.go]: We are planning to use /tmp/meta directory for saving meta temporarily.

Comment (stjohnjohnson) [inline meta.go]: It might be best to consolidate to `/sd/meta`

Comment (tk3fftk) [inline meta.go]: OK. I'll fix it.

**Comment by catto** [comment]
👍

**Comment by tk3fftk** [comment]
Thank you for the review!
And I try to use glide🐐

**Pull request merged**